lib/client.c

```diff
@@ -1600,6 +1643,16 @@ int64 STDCALL snowflake_affected_rows(SF_STMT *sfstmt) {
         /* no way to set the error other than return value */
         return ret;
     }
+
+    // Since Arrow isn't yet supported, we throw an error
+    if (is_force_arrow_mode(sfstmt->connection)) {
+        SET_SNOWFLAKE_STMT_ERROR(&sfstmt->error, SF_STATUS_ERROR_ATTEMPT_TO_RETRIEVE_FORCE_ARROW,
+                                 "Affected row-count not available using force-arrow results type", "",
+                                 sfstmt->sfqid);
+
+        return ret;
+    }
+
```

Comment:
+1

Reply:
What's +1? I assume you mean 'and this too,' in reference to some other comment. But I can't find the other comment. :)

Reply:
Oh I mean I like this piece of code haha

Reply:
:)